app/models/spouse.rb

```diff
@@ -11,6 +11,7 @@
 #  person_id  :bigint
 #
 class Spouse < Person
+  include Hashid::Rails
```

Comment:
Let's add some action on Spouses and test:

- Run action on index selecting 1 record
- Run action on index selecting all record from page
- Run action on index selecting all record from all pages
- Run action from show

We should also check if actions from [custom controls](https://docs.avohq.io/3.0/customizable-controls.html) work with custom IDs

Reply:
The action should modify something on the record